cmd/run/run.go

```diff
@@ -483,6 +483,7 @@ func RunServer(ctx context.Context, config *Config) error {
 		grpc_validator.UnaryServerInterceptor(),
 		otelgrpc.UnaryServerInterceptor(otelgrpc.WithTracerProvider(tp)),
 		middleware.NewRequestIDInterceptor(logger),
+		middleware.NewStoreIDInterceptor(),
```

Comment:
This needs to move above the `otelgrpc` middleware so that the `store` attribute in the trace is available before the top-level spans.

Reply:
nvm - I'm wrong.